src/content/plots/d3/HistogramD3.tsx

```diff
@@ -1,8 +1,15 @@
 import * as d3 from 'd3'
 
-const create = (el, layout, x: number[]) => {
-    // const start = Date.now();
+const config = {
```

Comment:
This seems to be the same as in ContoursD3.tsx. If they have to be the same, it might be worth extracting it into a constants.ts file.

Reply:
Good catch, I've made a new constants.ts file to keep this in, that HistogramD3.tsx and ContoursD3.tsx now import from